type_description_interfaces/srv/GetTypeDescription.srv

```diff
@@ -0,0 +1,42 @@
+# ROS interface type name, in PACKAGE/NAMESPACE/TYPENAME format
+string type_name
+
+# REP-2011 RIHS hash string, which looks like RIHSxx_hashvalue
+string type_version_hash
+
+# Whether to return the raw idl or msg file(s) in the response, if available
+bool include_raw_description true
+---
+# include_raw_description requested, and provided successfully
+uint8 RAW_DESCRIPTION_REQUESTED_PROVIDED = 0
+# include_raw_description disabled, so type_description_raw was left empty
+uint8 RAW_DESCRIPTION_NOT_REQUESTED_NOT_PROVIDED = 1
+# include_raw_description requested, but no raw version available (perhaps for )
+uint8 RAW_DESCRIPTION_REQUESTED_NOT_AVAILABLE = 2
+
+# True if the type description information is available and populated in the response
```

Comment:
Just to be extra pedantic here, we might want to say that if `successful` is False, all fields below are considered undefined except for `reason`.